lms/templates/courseware/info.html

```diff
@@ -81,6 +81,23 @@ <h3 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=cours
             </div>
           % endif
 
+          % if upgrade_banner:
+              <div class="upgrade-banner layout-1q3q layout-reversed">
+                <div class="upgrade-icon">
+                  <img src="${STATIC_URL}images/edx-verified-mini-cert.png">
+                </div>
+                <div class="upgrade-msg">
+                  <h4>${_("Give yourself an additional incentive to complete")}</h4>
+                  <p>${_("Earn a verified certificate.")}
```

Comment:
Put a class on the p-tag or a-tag so that we can use it for a click handler to fire the event.